packages/core/src/Application.ts

```diff
@@ -29,21 +29,7 @@ export class Application {
     return this;
   }
 
-  /**
-   *
-   * @deprecated Use boot instead
-   */
-  bootSync() {
-    this.providers.forEach(provider => {
-      provider.boot(this);
-    });
-
-    this.hasBooted = true;
-
-    return this;
-  }
-
-  async boot() {
+  boot() {
```

Comment:
While boot has no need to be async as of now, it *will* need to be async when ServiceProvider boot is turned into async. Consider this a preparation step. As such boot needs to be remain async , and all uses of boot needs to be awaited.

Reply:
I have re-implemented the asynchronous boot. Could you please review it again? 👀